runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -934,7 +979,7 @@ J9::Z::TreeEvaluator::inlineStringCodingHasNegativesOrCountPositives(TR::Node *n
    generateS390LabelInstruction(cg, TR::InstOpCode::label, node, processMultiple16CharsStart);
 
    // Load bytes and search for out of range character
-   generateVRXInstruction(cg, TR::InstOpCode::VL, node, vInput, generateS390MemoryReference(inputPtrReg, TR::Compiler->om.contiguousArrayHeaderSizeInBytes(), cg));
+   generateVRXInstruction(cg, TR::InstOpCode::VL, node, vInput, generateS390MemoryReference(inputPtrReg, 0, cg));
```

Comment:
I think this may have been a bug where we were repeatedly adding the header size after each iteration to the source register. Thanks for correcting it.

Reply:
I think it was OK, `inputPtrReg`  is evaluated to contain the byteArray and we add offset to that reference before we enter the loop. When you enter the loop, you always need to adjust the arrayHeader to get to the right element which is embedded in to memref - so I think it was fine for normal world,